### ISSUE TITLE
Show indication of unanswered questions quantity

### ISSUE THREAD

**Issue by talbacha** [DinisCruz/Maturity-Models#228]
Add quick link or shortcuts to show and jump to the remaining unanswered (pending) activities questions.

**Comment by DinisCruz** [comment]
hi @talbacha , you mean the questions with no values?

**Comment by talbacha** [comment]
Yes, as this will help the one who is answering the question to keep track of what is pending .. and jump directly to those questions.
